app/scripts/react-directives/device-manager/scan/common.jsx

```
@@ -56,6 +57,21 @@ const DeviceNameLabel = ({ title, selectable, selected, onSelectionChange, disab
 
+const InBootloaderErrorTag = ({ bootloaderMode, selectable }) => {
+  const { t } = useTranslation();
+  if (!bootloaderMode) {
+    return null;
+  }
+  if (selectable) {
+    return <ErrorTag text={t('scan.labels.in-bootloder')} />;
+  }
+  return (
+    <ErrorTag>
+      <Trans i18nKey={'scan.labels.in-bootloader-link'} components={[<a></a>]} />
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)